2016-02-26 17:32:55.484 [error] <0.349.0> CRASH REPORT Process 'Elixir.ExometerReportPhoenixChannel' with 0 neighbours crashed with reason: bad argument in call to ets:lookup('Elixir.ExometerPhoenixChannelDemo.PubSub', broadcast) in 'Elixir.Phoenix.PubSub':call/3 line 183
2016-02-26 17:32:55.498 [error] <0.373.0> CRASH REPORT Process 'Elixir.ExometerReportPhoenixChannel' with 0 neighbours crashed with reason: bad argument in call to ets:lookup('Elixir.ExometerPhoenixChannelDemo.PubSub', broadcast) in 'Elixir.Phoenix.PubSub':call/3 line 183
2016-02-26 17:32:55.509 [error] <0.382.0> CRASH REPORT Process 'Elixir.ExometerReportPhoenixChannel' with 0 neighbours crashed with reason: bad argument in call to ets:lookup('Elixir.ExometerPhoenixChannelDemo.PubSub', broadcast) in 'Elixir.Phoenix.PubSub':call/3 line 183
2016-02-27 12:39:30.936 [error] <0.344.0>@exometer_admin:handle_call:292 {new_entry,[socket,connections],guage,[],false} -*-> error:function_clause
[{exometer_admin,module,[guage],[{file,"src/exometer_admin.erl"},{line,544}]},{exometer_admin,exometer_default,2,[{file,"src/exometer_admin.erl"},{line,542}]},{exometer_admin,lookup_definition_,3,[{file,"src/exometer_admin.erl"},{line,502}]},{exometer_admin,lookup_definition,3,[{file,"src/exometer_admin.erl"},{line,466}]},{exometer_admin,handle_call,3,[{file,"src/exometer_admin.erl"},{line,271}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]
2016-02-27 12:39:30.936 [error] <0.341.0> CRASH REPORT Process <0.341.0> with 0 neighbours exited with reason: bad return value: {{exometer_core_app,start_phase,[preset_defaults,normal,[]]},{'EXIT',{function_clause,[{exometer_admin,new_entry,3,[{file,"src/exometer_admin.erl"},{line,175}]},{lists,foreach,2,[{file,"lists.erl"},{line,1337}]},{exometer_admin,load_predefined,0,[{file,"src/exometer_admin.erl"},{line,95}]},{application_starter,run_the_phase,4,[{file,"application_starter.erl"},{line,99}]},{application_starter,start_apps,3,[{file,"application_starter.erl"},{line,59}]},{application_starter,start,3,[{file,"appl..."},...]},...]}}} in application_master:init/4 line 134
2016-02-27 12:42:06.571 [error] <0.344.0>@exometer_admin:handle_call:292 {new_entry,[socket,connections],fast_counter,[],false} -*-> error:{required,function}
[{exometer_admin,handle_call,3,[{file,"src/exometer_admin.erl"},{line,286}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]
2016-02-27 12:42:06.571 [error] <0.341.0> CRASH REPORT Process <0.341.0> with 0 neighbours exited with reason: bad return value: {{exometer_core_app,start_phase,[preset_defaults,normal,[]]},{'EXIT',{{required,function},[{exometer_admin,new_entry,3,[{file,"src/exometer_admin.erl"},{line,175}]},{lists,foreach,2,[{file,"lists.erl"},{line,1337}]},{exometer_admin,load_predefined,0,[{file,"src/exometer_admin.erl"},{line,95}]},{application_starter,run_the_phase,4,[{file,"application_starter.erl"},{line,99}]},{application_starter,start_apps,3,[{file,"application_starter.erl"},{line,59}]},{application_starter,start,3,[{file,"..."},...]},...]}}} in application_master:init/4 line 134
2016-02-27 12:58:56.881 [error] <0.388.0> CRASH REPORT Process <0.388.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:58:56.900 [error] <0.390.0> CRASH REPORT Process <0.390.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:58:56.901 [error] <0.391.0> CRASH REPORT Process <0.391.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:58:56.901 [error] <0.392.0> CRASH REPORT Process <0.392.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:58:56.902 [error] <0.393.0> CRASH REPORT Process <0.393.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:58:56.902 [error] <0.394.0> CRASH REPORT Process <0.394.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:59:01.870 [error] <0.396.0> CRASH REPORT Process <0.396.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:59:01.870 [error] <0.397.0> CRASH REPORT Process <0.397.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:59:01.871 [error] <0.398.0> CRASH REPORT Process <0.398.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:59:01.871 [error] <0.399.0> CRASH REPORT Process <0.399.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:59:01.871 [error] <0.400.0> CRASH REPORT Process <0.400.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 12:59:01.872 [error] <0.401.0> CRASH REPORT Process <0.401.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:01:33.751 [error] <0.502.0> CRASH REPORT Process <0.502.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:01:33.757 [error] <0.503.0> CRASH REPORT Process <0.503.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:01:33.757 [error] <0.504.0> CRASH REPORT Process <0.504.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:01:33.758 [error] <0.505.0> CRASH REPORT Process <0.505.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:01:33.758 [error] <0.506.0> CRASH REPORT Process <0.506.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:01:33.758 [error] <0.507.0> CRASH REPORT Process <0.507.0> with 0 neighbours exited with reason: no match of right hand value {error,not_found} in 'Elixir.ExometerPhoenixChannelDemo.StatChannel':join/3 line 5 in gen_server:init_it/6 line 352
2016-02-27 13:19:16.693 [error] <0.344.0>@exometer_admin:handle_call:292 {new_entry,[socket,connections],guage,[],false} -*-> error:function_clause
[{exometer_admin,module,[guage],[{file,"src/exometer_admin.erl"},{line,544}]},{exometer_admin,exometer_default,2,[{file,"src/exometer_admin.erl"},{line,542}]},{exometer_admin,lookup_definition_,3,[{file,"src/exometer_admin.erl"},{line,502}]},{exometer_admin,lookup_definition,3,[{file,"src/exometer_admin.erl"},{line,466}]},{exometer_admin,handle_call,3,[{file,"src/exometer_admin.erl"},{line,271}]},{gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,629}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,661}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,240}]}]
2016-02-27 13:19:16.694 [error] <0.341.0> CRASH REPORT Process <0.341.0> with 0 neighbours exited with reason: bad return value: {{exometer_core_app,start_phase,[preset_defaults,normal,[]]},{'EXIT',{function_clause,[{exometer_admin,new_entry,3,[{file,"src/exometer_admin.erl"},{line,175}]},{lists,foreach,2,[{file,"lists.erl"},{line,1337}]},{exometer_admin,load_predefined,0,[{file,"src/exometer_admin.erl"},{line,95}]},{application_starter,run_the_phase,4,[{file,"application_starter.erl"},{line,99}]},{application_starter,start_apps,3,[{file,"application_starter.erl"},{line,59}]},{application_starter,start,3,[{file,"appl..."},...]},...]}}} in application_master:init/4 line 134
2016-02-28 18:01:53.693 [error] <0.361.0> gen_server exometer_report terminated with reason: no function clause matching random:uniform([result]) line 118
2016-02-28 18:01:53.693 [error] <0.361.0> CRASH REPORT Process exometer_report with 0 neighbours exited with reason: no function clause matching random:uniform([result]) line 118 in gen_server:terminate/7 line 826
2016-02-28 18:01:53.694 [error] <0.358.0> Supervisor exometer_core_sup had child undefined started with exometer_report:start_link() at <0.361.0> exit with reason no function clause matching random:uniform([result]) line 118 in context child_terminated
2016-02-28 18:04:02.032 [error] <0.361.0> gen_server exometer_report terminated with reason: {'module could not be loaded',[{'ExostatHelper',random,[[result]],[]},{exometer_function,get_value,4,[{file,"src/exometer_function.erl"},{line,242}]},{exometer,get_value,2,[{file,"src/exometer.erl"},{line,315}]},{exometer_report,get_values,2,[{file,"src/exometer_report.erl"},{line,1366}]},{exometer_report,do_report,2,[{file,"src/exometer_report.erl"},{line,1152}]},{exometer_report,handle_report,4,[{file,"src/exometer_report.erl"},{line,1139}]},{exometer_report,handle_info,2,[{file,"src/exom..."},...]},...]}
2016-02-28 18:04:02.032 [error] <0.361.0> CRASH REPORT Process exometer_report with 0 neighbours exited with reason: call to undefined function 'ExostatHelper':random([result]) in gen_server:terminate/7 line 826
2016-02-28 18:04:02.032 [error] <0.358.0> Supervisor exometer_core_sup had child undefined started with exometer_report:start_link() at <0.361.0> exit with reason call to undefined function 'ExostatHelper':random([result]) in context child_terminated
2016-02-28 18:05:21.618 [error] <0.243.0> gen_server exometer_report terminated with reason: no function clause matching exometer_function:'-get_value/4-lc$^1/1-0-'(0.5194077573330378) line 242
2016-02-28 18:05:21.618 [error] <0.243.0> CRASH REPORT Process exometer_report with 0 neighbours exited with reason: no function clause matching exometer_function:'-get_value/4-lc$^1/1-0-'(0.5194077573330378) line 242 in gen_server:terminate/7 line 826
2016-02-28 18:05:21.618 [error] <0.240.0> Supervisor exometer_core_sup had child undefined started with exometer_report:start_link() at <0.243.0> exit with reason no function clause matching exometer_function:'-get_value/4-lc$^1/1-0-'(0.5194077573330378) line 242 in context child_terminated
